<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSettingDialog</class>
 <widget class="QDialog" name="MSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>色差公式算法</string>
   </property>
   <widget class="QRadioButton" name="space4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lab Plus (最好,最慢)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="space1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB Plus (推荐的)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="space3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>115</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lab</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="space2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>48</y>
      <width>115</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HSV</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="space0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>115</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>色阶模式</string>
   </property>
   <widget class="QRadioButton" name="type0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>彩色模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="type1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>灰阶模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="type2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>黑白模式</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>平面/立体地图画</string>
   </property>
   <widget class="QRadioButton" name="plane">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>平面模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成高度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="height">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="cube">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>立体模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>生成高度区间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mheight1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>到</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mheight2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>其他设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>选择背景颜色</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dither">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>启用抖动算法</string>
    </property>
   </widget>
   <widget class="QToolButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成起始点坐标 (地图左上角)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X坐标</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="beginX">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Z坐标</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="beginZ">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="edition">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏版本</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bedrock">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>基岩版BE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="java">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Java版JE
(暂不可用)</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="autoButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>620</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>默认设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>620</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>1151</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>色差公式理论上Lab Plus为最佳，也是当今最专业运用最广泛的算法，但速度较慢
RGB Plus算法综合效果较好，当图片较大时我们更推荐它
其他公式为对应颜色空间的欧氏距离算法，相对不符合人眼视觉会有较大色差，不建议使用

色阶中彩色为使用MC中能用上的六十几种地图基色
灰阶是只使用R、G、B相同的地图基色
黑白只使用黑白羊毛

抖动算法是一个效果很棒的像素画优化算法，在大分辨率图片上效果更为明显
但它是分辨率换色阶的效果，因此图片分辨率超过256*256像素时，我们强烈建议启用抖动
而低于此分辨率时，可自行对比预览图决定是否启用抖动

背景颜色是png等图片带有alpha透明通道时对半透明底色的叠加算法，默认为黑色

游戏生成区间在-60~319（基于超平坦世界），若是1.17之前版本，请设置为5~255，或根据地图自身需求调整

选定地图起始点坐标，一般X与Z加64后均为128倍数

Java版、以前的游戏版本方块、语言切换和schematic格式文件以及界面美化功能优化等功能工程量较大，仍在开发中，暂不可用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
